slsim/Sources/__init__.py

```diff
@@ -0,0 +1,13 @@
+from .galaxies import Galaxies
```

Comment:
Does importing all of these at the package level slow the import time in some cases?

Reply:
I wouldn't expect this to slow things considerably and doubt this is the biggest runtime bottleneck in the code, but can run a profiler with and without submodule imports if you'd prefer?

Reply:
Either aliasing or specifiying in __all__ for the submodule is necessary to uphold linter requirements from ruff. Missed these when switching to __all__ instead of aliasing, thanks for the catch!

Reply:
It would be good to check!

Reply:
I used the python importtime profiler + tuna for visualization to profile a simple script which only contains imports. In the case of original.log, this script contains all imports in LensPop for the case of galaxy-galaxy strong lensing, while refactor.log contains the imports in this PR's lens_pop.py.

You can visualize the logs by running `pip install tuna` and `tuna original.log`. This simple profiling run shows that the original setup takes 2.301 s while the refactored imports take 1.571 s, including overhead from the profiler itself.

[original.log](https://github.com/user-attachments/files/16638199/original.log)
[refactor.log](https://github.com/user-attachments/files/16638200/refactor.log)

Reply:
Thank you very much @jacob-hjortlund ! This is very good to know!